krm-functions/README.md

```diff
@@ -0,0 +1,3 @@
+# KRM functions
+
+This directory contains in-tree SIG-CLI sponsored and authored KRM functions.
```

Comment:
It seems this file should live in `krm-functions/sig-cli`?

Reply:
Ah yes, thanks for the catch. Will move it.